### PR TITLE
Use Company Name of Table Information

### PR DESCRIPTION
**Pull request by pri-kise**
Fixes the Drilldown of the Table Information Page to use the correct Company Name

### CODE REVIEW

File: Modules/System/Table Information/src/TableInformation.Page.al [hunk 1]
```diff
@@ -102,4 +102,12 @@ page 8700 "Table Information"
             Rec.SetFilter("Company Name", '%1|%2', '', CompanyName);
         FilterGroup(0);
     end;
+
+    local procedure GetTableUrl(TableInformation: Record "Table Information"): Text
+    begin
+        if TableInformation."Company Name" = '' then
```
[the remark this answers is not in the document]
Comment (pri-kise) [reply Modules/System/Table Information/src/TableInformation.Page.al]: I updated it accordingly

**Comment by mazhelez** [comment]
@pri-kise Can you please also add a description to the pull request?

**Comment by mazhelez** [comment]
> > @pri-kise Can you please also add a description to the pull request?
> 
> Is this okay for you?

Yep :) All good.

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 42366.

**Comment by pri-kise** [comment]
@JesperSchulz I like it that this issue is fixed now in the system application, but it's a little bit disappointing for community contributions if they are only closed.
Maybe you will find some time in the near futureto transform this repository into a real product repository instead of a complicated mirror of microsoft's real repository.

**Comment by JesperSchulz** [comment]
This one should have gotten merged. Will do so. The issue is automatically closed, when our internal work items are closed - and while that is fine for issues, it's wrong for PRs. Will reopen and merge :-)
But to your other point: we're working on that! I hope that we will move our real repository out on GitHub within the next half year 🥳

**Review by JesperSchulz — APPROVED** [review]

**Comment by JesperSchulz** [comment]
Changes will ship with 20.4

**Pull request merged**